subjects/java/checkpoints/single-linkedlist/README.md

```diff
@@ -0,0 +1,119 @@
+## Single Linked List
+
+### Instructions
+
+Create a single linked list data structure that implements the following methods:
+
+- `at(int index)`: to access an element by its index, if the index is out of bound return -1.
+- `add(int value)`: to add an element at the end of the list.
+- `remove(int index)`: to remove an element by its index.
```

Comment:
What should happen if the index is out of bounds? 
What should happen for a negative index? Should `-1` refer to the lats item, `-2` to the second last ...?

Reply:
calling at with -1/-2: will return -1
calling remove with -1/-2: nothing will be removed the index not found.

it's already defined in the subject.